libres/lib/enkf/enkf_plot_gendata.cpp

```diff
@@ -147,28 +147,21 @@ static void enkf_plot_gendata_reset(enkf_plot_gendata_type *plot_gendata,
 }
 
 void enkf_plot_gendata_load(enkf_plot_gendata_type *plot_data, enkf_fs_type *fs,
-                            int report_step,
-                            const bool_vector_type *input_mask) {
+                            int report_step) {
 
     state_map_type *state_map = enkf_fs_get_state_map(fs);
     int ens_size = state_map_get_size(state_map);
-    bool_vector_type *mask;
-
-    if (input_mask)
-        mask = bool_vector_alloc_copy(input_mask);
-    else
-        mask = bool_vector_alloc(ens_size, false);
-
-    state_map_select_matching(state_map, mask, STATE_HAS_DATA, true);
 
+    std::vector<bool> mask =
```

Comment:
As Joakim suggested other places:
 
```suggestion
    const auto& mask =
```